static/styles/subscriptions.css

```diff
@@ -603,21 +576,6 @@ h4.stream_setting_subsection_title {
     margin-bottom: 10px;
 }
 
-.stream-creation-body {
-    section.block {
-        margin-bottom: 20px;
-    }
-
-    #announce-new-stream {
-        margin-top: 10px;
-
-        div[class^="fa"] {
-            margin-left: 3px;
-            margin-right: 8px;
-        }
-    }
-}
-
```

Comment:
Added this CSS in the nested CSS for `.stream-creation-body` below in the file to avoid duplicity in written CSS.